core/frontend/src/CheckpointConnection.ts

```diff
@@ -253,14 +241,18 @@ export class RemoteBriefcaseConnection extends CheckpointConnection {
    * @throws [[IModelError]] if the IModelConnection is read-only or there is a problem updating the extents.
    */
   public async updateProjectExtents(newExtents: AxisAlignedBox3d): Promise<void> {
-    return this.editing.updateProjectExtents(newExtents);
+    if (OpenMode.ReadWrite !== this.openMode)

```

Comment:
Just noticed RemoteBriefcaseConnection has this but BriefcaseConnection doesn't - we should probably add it (in separate PR).

Reply:
I will do that.